contrib/init/systemd/docker.service

```diff
@@ -1,7 +1,7 @@
 [Unit]
 Description=Docker Application Container Engine
 Documentation=https://docs.docker.com
-After=network.target docker.socket firewalld.service
+After=network-online.target docker.socket firewalld.service
```

Comment:
I don't understand how to do git rebase. I have read contributors guide and @GordonTheTurtle 's advises and a lot of articles in Internet about it. I catch error "fatal: Needed a single revision
invalid upstream" and don't know what to do with this. It's too hard for me. Sorry :-( 
Please let @JamieGarside make his own pull request. I closed this PR.